engine/class_modules/sc_shaman.cpp

```diff
@@ -6803,6 +6813,14 @@ struct lightning_bolt_t : public shaman_spell_t
         p()->buff.stormkeeper->decrement();
       }
       p()->sk_during_cast = false;
+
+      if ( p()->is_ptr() )
+      {
```

Comment:
I know this can be condensed into a singular conditional, but I think when it comes time to prune all these after the PTR lands that this will be cleaner to diff.